docs/loading/S3_Load.md

```diff
@@ -0,0 +1,272 @@
+# Load data from Amazon S3
+
+StarRocks provides two options for loading data from S3:
+
+1. Asynchronous loading using Broker Load (`WITH BROKER`)
```

Comment:
Will `WITH BROKER` connote that brokers are required for Broker Load, but actually the answer is "No"? (From v2.5 users do not need to deploy brokers, and yet `WITH BROKER` is retained in the statement, but users do not need to specify any broker-related parameters.)

Reply:
@amber-create I included with broker in the text because it is the text that the reader needs to use. (Just like I used files() in the other section). I realize that if the reader wants to use the local file broker they have to install that, but I would leave that out of this doc and only talk about that in a page about loading from local files.

Are you suggesting that the heading end at Broker Load?

Reply:
Yeah, I think it's better to remove "(`WITH BROKER`)" and make the sentence end at "Broker Load". Do you think that's a good idea?